config/manager/kustomization.yaml

```diff
@@ -1,2 +1,8 @@
 resources:
 - manager.yaml
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+images:
+- name: controller
+  newName: quay.io/opentelemetry/opentelemetry-operator
+  newTag: v0.27.0
```

Comment:
This was recommended by the migration/upgrade guide and without it I could not get the bundle to use the operator image. It would try to find an image called `controller` which would obviously absent.

Reply:
There is a target in the Makefile that sets the image name to what we have as vars there:

https://github.com/open-telemetry/opentelemetry-operator/blob/aaabe4ef12e9e445c165f19385dfc5b431c781ef/Makefile#L69-L71

But if they do have a new recommendation about this, we should definitely follow it.